deploy-agent/WORKSPACE

```diff
@@ -0,0 +1,13 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+http_archive(
+    name = "rules_python",
+    sha256 = "a30abdfc7126d497a7698c29c46ea9901c6392d6ed315171a6df5ce433aa4502",
+    strip_prefix = "rules_python-0.6.0",
+    url = "https://github.com/bazelbuild/rules_python/archive/0.6.0.tar.gz",
```

Comment:
rules_python now has 0.21.0 https://github.com/bazelbuild/rules_python/releases  shall we use the latest ?

Reply:
sg, let me try it.

Reply:
Tried with the latest version but had some issues. Let's use the current version for now, which was released 01/2022.